btk/measure.py

```diff
@@ -209,6 +250,7 @@ def __iter__(self):
     def run_batch(self, batch, index, **kwargs):
         """Perform measurements on a single blend."""
         output = []
+        surveys = kwargs.get("surveys", 0)
```

Comment:
Should `surveys` should be a required argument or an attribute for the `MeasureGenerator`?